lib/simplecov/source_file.rb

```diff
@@ -199,9 +199,8 @@ def restore_ruby_data_structure(structure)
       # put them through here.
       return structure if structure.is_a?(Array)
 
-      # as of right now the keys are still symbolized
```

Comment:
The docs on top of this method also talk about key symbolization so should probably be updated too?

Reply:
Great catch!